website/src/docs/migration-guides.md

```diff
@@ -186,6 +188,11 @@ Remove `rotateImage`, `protect`, and `canvasToBlob` from the plugin prototype. T
 
 To migrate: use exposed options only.
 
+### Known issues
+
+* [`ERESOLVE could not resolve` on npm install](https://github.com/transloadit/uppy/issues/4057).
+* [@uppy/svelte reports a broken dependency with the Vite bundler](https://github.com/transloadit/uppy/issues/4069).
```

Comment:
nit: I think it's nicer to inline the problem and solution here and optionally a link to the issues.

Reply:
What solutions though?

Reply:
ah I thought there were workarounds